its/projects/shared/struts-1.3.9-diet/core/pom.xml

```diff
@@ -25,7 +25,7 @@
     <parent>
         <groupId>org.apache.struts</groupId>
         <artifactId>struts-parent</artifactId>
-        <version>1.3.9</version>
+        <version>1,3,9</version>
```

Comment:
Personally, I like to keep ITs looking like "real" projects. Here we are testing a corner case, that has no dependency on the SonarQube version, so I would go for an "invoker" test.